Wikipedia/Code/DiffFetcher.swift

```diff
@@ -40,6 +41,57 @@ class DiffFetcher: Fetcher {
         }
     }
     
+    func fetchWikitext(siteURL: URL, revisionId: Int, completion: @escaping (Result<String, Error>) -> Void) {
+        
+        let params: [String: AnyObject] = [
+            "action": "query" as AnyObject,
+            "prop": "revisions" as AnyObject,
+            "revids": "\(revisionId)" as AnyObject,
+            "rvprop": "content" as AnyObject,
+            "format": "json" as AnyObject
+        ]
```

Comment:
`performMediaWikiAPIGET` takes `[String: Any]` so these casts can be avoided. It looks like this was modeled after `PageHistoryFetcher` which takes `[String: AnyObject]` because it's a subclass of the obj-c `WMFLegacyFetcher`. It might be possible to update that to a `Fetcher` but that's outside the scope of this PR